### DISCUSSION TITLE
INET4.5.1: UdpBasicApp - simulation breaks in some PCs when sendInterval is set too low

### DISCUSSION THREAD

**Discussion by antoniogrilo** [omnetpp/omnetpp#1276] [General]
I have a scenario where I try to measure the maximum throughput of an IEEE 802.15.4 network. I configure UdpBasicApp as app[0] in two hosts, then I try to decrease the sendInterval parameter until the network capacity saturates. While in my laptop I get the intended result (approx. 150 kbit/s for the selected packet size), my students are getting a weird behavior: the packet rate starts to drop sharply at both sender and receiver. What could be the reason? Not enough memory? Incompatibility with newer microprocessors? 

I am attaching the scenario. The configuration of interest is the UdpBasicApp configuration.
[Ex1.zip](https://github.com/user-attachments/files/17982776/Ex1.zip)

**Comment by levy** [comment]
The simulation must produce the exact same results independently of the used hardware, operating system or compiler version. There's either some difference between the simulations you don't see or there's a serious bug. I think the latter is very unlikely because it would bite us more often. So please take a look and try to find differences between the two simulations.

If you are 100% sure that there are no differences in the INI/NED/C++ files and you still get different results, then please create an issue and I'll mark it as a bug.

**Reply by antoniogrilo** [reply]
Yes, I am pretty sure. Everybody is using the same example, in OMNET++ 6.1, INET 4.5.4. It happens both in Linux and Windows.

**Reply by levy** [reply]
Could you please create an issue in the [INET](https://github.com/inet-framework/inet) repository and describe the exact steps to reproduce the incorrect behavior? It's ok if the incorrect behavior only shows up occasionally or only on some computers, you can simply state this fact. Also, the issue should contain a narrowed down version of the NED and INI files needed to run the simulation. Please be as specific as possible, because from the above description, for example, it's unclear which statistic you are referring to and how you change the sendInterval parameter.
